MulledDockerfile

```diff
@@ -1,27 +1,37 @@
-FROM debian:latest
+FROM debian:10-slim as base
 
 #  $ docker build -f MulledDockerfile -t bioconda/mulled_container:latest
```

Comment:
@dpryan79, could you build and upload this to `quay.io/dpryan79/mulled_container:latest`?
In the next few days I'll create another repo that'll hold this and uploads to `bioconda` on Docker Hub, but in the meantime we can just update yours :).

Reply:
Sure, I'll try to get that done today

Reply:
Not sure if this is an issue, since `conda env list` still works:

```
$ docker run --rm -it dpryan79/mulled_container:latest /bin/bash

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.
```

Reply:
Weird, adding `conda init bash` doesn't change it, which surprises me a bit.

Reply:
I'll just push this then.

Reply:
Ah, balls. Hold on, I'll fix it right now.

Reply:
It's live

Reply:
Shouldn't be much of an issue, but can you rebuild from https://github.com/bioconda/bioconda-utils/pull/662/commits/8e6c9aa56684d54955d597f9f07683d66e31ac0a ?
Sorry for not catching this before!

Reply:
I'll have to reread https://www.gnu.org/software/bash/manual/html_node/Bash-Startup-Files.html  yet again, I guess -.-

Reply:
Well, the `/bin/bash -l` in the entrypoint was correct. But there were other very dump oversights I fixed in https://github.com/bioconda/bioconda-utils/pull/662/commits/7ef9e1ac84c09b2042ed83e841e77c9d968e647f .
Can you build for that commit again when you find the time for it? Thanks and sorry again.